src/importer/importer.py

```
@@ -6549,6 +6713,18 @@ def convert_record_types(record: dict, table_name: str) -> dict:
             converted[field_name] = ""
             continue
 
+        if (
+            table_name in _SK_STORAGE_TABLES
+            and field_name in _SK_BLANK_TEXT_FIELDS
+            and isinstance(value, str)
+            and not value.strip()
+        ):
+            # The official SK 産地名 span may be blank (initial value Ｓ). The v2
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `Ｓ` (FULLWIDTH LATIN CAPITAL LETTER S). Did you mean `S` (LATIN CAPITAL LETTER S)?

(RUF003)